src/gsi/gsi_rfv3io_mod.f90

```diff
@@ -1352,7 +1364,7 @@ subroutine read_fv3_netcdf_guess(fv3filenamegin)
          call GSI_BundleGetPointer ( GSI_MetGuess_Bundle(it), 'tv' ,ges_tv ,istatus );ier=ier+istatus
          call GSI_BundleGetPointer ( GSI_MetGuess_Bundle(it), 'q'  ,ges_q ,istatus );ier=ier+istatus
          call GSI_BundleGetPointer ( GSI_MetGuess_Bundle(it), 'oz'  ,ges_oz ,istatus );ier=ier+istatus
-         if (l_use_dbz_directDA .or. if_model_dbz) then
+         if (l_use_dbz_directDA .or. if_model_dbz .or. if_model_fed) then
```

Comment:
similarly,  could be "if (l_use_dbz_directDA .or. nphyvarsio > 0) then "

Reply:
@TingLei-NOAA 
Does l_use_dbz_directDA (true) mean to use the DBZ operator which is a function of hydrometeors and other variables? There is a description but I am still not sure its exactly meaning. 

src/gsi/directDA_radaruse_mod.f90
l_use_dbz_directDA          ! assimilation of radar reflectivity(dbz) data using direct DA capability

Reply:
@hongli-wang  Yes. I think so.  And other implications include (excerpt from @junpark217 :
```
If 'l_use_dbz_directDA' is set as true

run initialization subroutines for Z analysis       - e.g.) directDA_radaruse_mod.f90
read Z OBS from file 'dbzbufr'       - e.g.) read_obs.F90, read_radarref_mosaic_directDA.f90
set static BEC for hydrometeors if necessary       - e.g.) prewgt_reg.f90
read additional hydrometeors variables required for Z DA and conduct CV transform (power transform or log transform)
 - e.g.) cplr_get_fv3_regional_ensperts.f90, cplr_get_wrf_mass_ensperts.f90, cplr_read_wrf_mass_guess.f90, gsi_rfv3io_mod.f90, ....
use CAPS implemented routines in calculating H(xb) of Z and Jacobians        - e.g.) setupdbz.f90, intdbz.f90, stpdbz.f90
Update associated hydrometeor variables      - e.g.) update_guess.f90
When writing out the final analysis after DA, restore hydrometeor from PQ or logQ space into Q space. 
- e.g.) gsi_rfv3io_mod.f90, ./cplr_wrwrfmassa.f90
```
Hope this helps!

Reply:
> similarly, could be "if (l_use_dbz_directDA .or. nphyvarsio > 0) then "

@TingLei-NOAA 
Revised the code. Thanks.